src/dispdrivers/vga16controller.cpp

```diff
@@ -647,6 +647,67 @@ void VGA16Controller::rawCopyToBitmap(int srcX, int srcY, int width, void * save
 }
 
 
+void VGA16Controller::rawDrawBitmapWithMatrix_Mask(int originX, int originY, Rect & drawingRect, Bitmap const * bitmap, const float * invMatrix)
```

Comment:
all of the various function calls that draw a bitmap using a transform matrix _currently_ do not support the `saveBackground` argument that the non-transformed bitmap plots support.  this means that for now, they can't be used for sprite plotting.  this is why they all have a commented out line for "get pixel in row", as that is what is used to save the background area for sprites

supporting transformed sprites may be added in the future